addOns/alertFilters/src/main/java/org/zaproxy/zap/extension/alertFilters/GlobalAlertFilterParam.java

```diff
@@ -164,7 +164,7 @@ protected void parseImpl() {
                 alertFilters.add(
                         new AlertFilter(
                                 -1,
-                                sub.getInt(FILTER_RULE_ID_KEY),
+                                sub.getString(FILTER_RULE_ID_KEY),
```

Comment:
Is anything needed here to convert configs? Should the config be versioned?

Reply:
No, the property was a string already (the `getInt()` was doing the conversion).